examples/ios/swift/Migration/README

```diff
@@ -0,0 +1,29 @@
+The Migration project shows several examples of migrations and migration blocks.
+
+The purpose of this example is to provide a more in depth view of certain problems and pitfalls users might
+face when migrations get more complex and the number of versions increases.
+
+How to use the example:
+You can build and run the project since this is the default state (see
+`shouldCreateExamples` in `AppDelegate`).
+There is nothing much so see in terms of unit tests, console log or in the app itself.
+If there is no error the migration was successful.
+The files to look at are located in the `Examples` folder. Every file contains an extract
+of everything necessary for this version (schema version, objects, migration).
+
+How to create new examples:
+- Set `shouldCreateExamples` in `AppDelegate` to true.
+- Duplicate the most recent `Example_vx`.
+- Comment out the previous one. Update the current one:
+  - Add a new version to the `RealmVersion` enum.
+  - Update the `Object` definitions according to your new example.
+  - Update the migration block.
+  - Create some examples for that specific schema version (they will be used in later schema version).
+- Run the app.
+- The realm file (path printed to console) created by this run must then be added to project.
+
+Important notes:
+- Code in all `Example_vx` files except the most recent one needs to be commented out to avoid
```

Comment:
We shouldn't do anything with commenting out code, its not stable and there are more efficient ways to do it, i.e preprocessor flags

Reply:
Fixed with the `xcconfig` file.